_data/banking.yml

```
@@ -908,7 +908,7 @@ websites:
       - hardware
     doc: https://www.lansforsakringar.se/privat/att-vara-kund/digitala-tjanster-telefonbank/internetbanken/
 
-  - name: MandT Bank
+  - name: M&T Bank
```

I think it's better to use this encoding for HTML entities:
```suggestion
  - name: M&amp;T Bank
```